scripts/test.py

```diff
@@ -182,9 +180,13 @@ def render(args):
         test_permutations=test_permutations
     )
 
-    output_file = open(output_file, 'w')
-    output_file.write(rendered)
-    output_file.close()
+    if args.check:
+        with open(output_file, 'r') as file:
+            if file.read() != rendered:
+                fail('circle CI configuration was not rendered correctly')
```

Comment:
I'd prefer if this string said `circle CI configuration is out of date, re-render it`